libindy/src/domain/anoncreds/proof_request.rs

```diff
@@ -178,7 +181,8 @@ impl Validatable for ProofRequest {
         }
 
         for (_, requested_attribute) in value.requested_attributes.iter() {
-            if requested_attribute.name.is_empty() {
+            if (requested_attribute.name == None || requested_attribute.name == Some("".to_string())) &&
```

Comment:
```suggestion
            if (requested_attribute.map(str::is_empty).unwrap_or(true)) &&
```

Reply:
and the similar for vector below

Reply:
This is funny, how me, @Artemkaaas and you have three different points of view on this code.
I don't think that `map` and `unwrap_or` is more clear than current variant.
It needs to be thought about whilst equality checks are clear right from the start.

Reply:
I don't mind to to have separate statements for 2 cases, but we shouldn't allocate temporary empty string just to compare with the input